service.py

```diff
@@ -1,141 +1,137 @@
-from concurrent import futures
 import grpc
-import utility
 import hwsc_file_transaction_svc_pb2
 import hwsc_file_transaction_svc_pb2_grpc
+import utility
 from enum import Enum
+from concurrent import futures
 
 
-class State(Enum):
-    """A current state class of file transaction service"""
-    AVAILABLE = 0
-    UNAVAILABLE = 1
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+    """A FileTransactionService class contains services for handling file transactions."""
 
+    def __init__(self, server):
+        self.__server = server
```

Comment:
server here is the same Server at line https://github.com/hwsc-org/hwsc-file-transaction-svc/pull/44/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR5